DPRHTML/search.html

```diff
@@ -24,7 +24,7 @@
             searchType = parseInt(parameterSections[1], 10);
             break;
           case 'query':
-            searchString = parameterSections[1];
+            searchString = decodeURIComponent(parameterSections[1]);
```

Comment:
oh wow!

Reply:
this is of course a short term fix. there will be an item on you for post-alpha to unify xul and web and as part of that we need to get rid of this change.